components/timeline/TimelineDiscover.vue

```diff
@@ -18,6 +18,13 @@ function shorten(str: string, maxLen: number): string {
 function updateUTM(url: string): string {
   return url.replace('pocket-newtab', 'mozilla')
 }
+
+const clipboard = useClipboard()
```

Comment:
not having to import this still throws me off so much coming from `Web` and `web-client` land :')

Reply:
I miss explicit imports soooo bad